### DISCUSSION TITLE
Query things that have "no role"

### DISCUSSION THREAD

**Discussion by iiLearner** [testing-library/eslint-plugin-testing-library#521] [Q&A]
Hi, How do I query in DOM items that do not have a role? for exampe:

```ts
<div className={classNames('ios', styles.ff_ar_choice_modal_content__platform)}>
	<PlatformButton onClick={() => onPlatformSelect(PLATFORMS.iOS)}>
		<AppleIcon />
	</PlatformButton>
</div>
```

I'm trying to query this and then click it but since its a simple div with an image inside,  im not sure how to query it.

**Comment by Belco90** [comment]
Hi! I don't get why are you interested in clicking on the div in particular. You have a button inside, you should query that element and click on it instead.

Am I missing something else?

**Comment by iiLearner** [comment]
Hi, that's not a button, its another div element (named as button) which simply contains a svg icon.

**Reply by Belco90** [reply]
That's really misleading. If it's an interactive element, it shouldn't be a div but a proper HTML element. In this case, it seems it should be an actual button.
